backend/src/main/java/com/github/lukaskraus/backend/service/CarService.java

```diff
@@ -17,6 +17,10 @@ public List<Car> getAllCars() {
         return carRepo.findAll();
     }
 
+    public Car getCarById(String id) {
+        return carRepo.findById(id).orElseThrow();
```

Comment:
```suggestion
        return carRepo.findById(id).orElseThrow(() -> new NoSuchElementException("Car with id \"" + id + "\" not found"));
```

damit klar erkennbar ist was für eine Exception geworfen wird mit message